lib/evaluateArithmetic.js

```diff
@@ -13,6 +13,15 @@ function evaluateArithmeticDFS(node) {
   let innerNodeStatus;
   if (NodeType.isParenthesis(node)) {
     innerNodeStatus = evaluateArithmeticDFS(node.content);
+    if (innerNodeStatus.hasChanged()) {
```

Comment:
don't have to do this anymore cause it should be handled before this function is called